deployer/README.md

```diff
@@ -1175,3 +1175,260 @@ the first created. You can check if the route in question is defined in multiple
     logging     kibana-ops   kibana-ops.example.com              logging-kibana-ops
 
 (In this example there are no overlapping routes.)
+
+## Splitting your Fluentd for high scale capacity (Advanced)
```

Comment:
Nice, though this seems like a lot of steps, and providing an ansible playbook which does all that is described here, just adding a set of variables, would be really powerful.